src/presentation/http/router/auth.test.ts

```diff
@@ -0,0 +1,76 @@
+import { describe, test, expect } from 'vitest';
+
+describe('Auth API', () => {
+  describe('POST /auth', () => {
+    test('Returns 401 when session is not valid', async () => {
+      const expectedStatus = 401;
+      const refreshToken = 'not-validToken';
```

Comment:
From the variable name it is not quite clear that this is a invalid token. The data in the variable should be more realistic - the character set that makes the token invalid.